### ISSUE TITLE
port over better k space fitter from MOF

### ISSUE THREAD

**Issue by esheldon** [esheldon/ngmix#55]

**Comment by beckermr** [comment]
Is this a vs item?

**Comment by esheldon** [comment]
the MOF code is gmix v1 so we can't do it right now.  But might be good to do after v2 is out

**Comment by beckermr** [comment]
We probably need to port fitvd/shredder to v2 BTW.

**Comment by esheldon** [comment]
yes; there is no hurry for that, we won't need to run that code again until its needed for the deep field work

**Comment by beckermr** [comment]
Unless we run it in mdet I suppose.
